src/components/Calender/DateDisplay.tsx

```diff
@@ -29,7 +29,7 @@ export const DateDisplay = (props) => {
 
     const populateTasks = (date: Date): Object[] => {
         const tasks = useSelector((state) => {
```

Comment:
up to you, i'm always a fan of making changes now since a lot of times it is forgotten later. the more you push things to later the more your tech debt compounds